src/views/webview-app/connection-form.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import CompassConnectionForm from '@mongodb-js/connection-form';
 import { Modal, css, spacing } from '@mongodb-js/compass-components';
 import { v4 as uuidv4 } from 'uuid';
+import type { ConnectionInfo } from 'mongodb-data-service-legacy';
```

Comment:
Unfortunately, we will have to continue using ConnectionInfo from mongodb-data-service-legacy until we add OIDC.